plugins/modules/files/iso_customize.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022, Ansible Project
+# Copyright (c) 2022, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: iso_customize
+short_description: Add/remove/change files in ISO file
+description:
+  - This module is used to add/remove/change files in ISO file.
+  - We can change the file with combinded option "delete_files" (delete the file inside ISO)
```

Comment:
> I am not very fond of software that, in order to do X, instructs you to do A and then B. Instead of writing a module that "adds" and "deletes", but if you want to "change" you must "add" and "delete", I think it would make more sense to have a state parameter, with possible values "present", "absent" and "replace". Again, just my personal preference.

I'm studying your point carefully....   Thanks very much.

Reply:
@ZouYuhua I have to agree with you there, even though I still feel this is a bit of an odd interface. Will drop another comment to you later if I think of an alternative, but for now I think we're good on that.

Reply:
@russoz 
Thanks very much.  I will change it ASAP when I get another comment from you to request to change.